crates/forge/tests/e2e/common/runner.rs

```diff
@@ -56,6 +59,33 @@ pub(crate) fn setup_package(package_name: &str) -> TempDir {
     setup_package_with_file_patterns(package_name, BASE_FILE_PATTERNS)
 }
 
+pub(crate) fn setup_package_with_rpc_url_placeholders(
+    package_name: &str,
+    file_patterns: &[&str],
+) -> TempDir {
+    let temp_dir = setup_package_with_file_patterns(package_name, file_patterns);
+    replace_rpc_url_placeholders(temp_dir.path());
+    temp_dir
+}
+
+fn replace_rpc_url_placeholders(dir_path: &Path) {
+    let url = node_url_with_version().unwrap();
```

Comment:
i think this should be smarter and only replace the files with the explicit `.cairo.template` suffix. can be done in later PR

Reply:
I don't see a need, what it replaces is not a valid syntax for our attributes anyways

Reply:
i don't understand this response @piotmag769, can you explain?

Reply:
I meant if the stuff in not replaced in `.cairo` files we get an error in compilation anyways, so idk if the function being smarter really makes a lot of sense